go.mod

```diff
@@ -44,3 +44,5 @@ require (
 )
 
 replace github.com/gorilla/rpc v1.2.0+incompatible => github.com/gorilla/rpc v1.2.0
+
+replace github.com/minio/minio => ../../minio/minio
```

Comment:
Leaving a comment here to remind us to change this line once the minio server PR is merged.